cocos/asset/assets/effect-asset.jsb.ts

```diff
@@ -38,5 +44,148 @@ effectAssetProto._ctor = function () {
     this.hideInEditor = false;
 };
 
+export declare namespace EffectAsset {
```

Comment:
Please add comments that why need to declare again in .jsb.ts too.

Reply:
updated